core-tests/src/test/java/org/mule/runtime/core/DefaultMessageContextTestCase.java

```diff
@@ -105,82 +80,11 @@ public void overrideCorrelationIdInContext() {
   }
 
   @Test
-  public void parentOverrideCorrelationIdInContext() {
-    final MuleMessage messageParent = MuleMessage.builder().payload(TEST_PAYLOAD).build();
-    final MuleMessage message = MuleMessage.builder().payload(TEST_PAYLOAD).build();
-    final MuleEvent eventParent = MuleEvent.builder(executionContextWithCorrelation).message(messageParent).flow(flow).build();
-    ((DefaultFlowCallStack) eventParent.getFlowCallStack()).push(new FlowStackElement(flow.getName(), "/0"));
-    final MuleEvent event = MuleEvent.builder(executionContextWithCorrelation).message(message).flow(flow).build();
-    ((DefaultMuleEvent) event).setLegacyCorrelationId(CORRELATION_ID);
-    ((DefaultMuleEvent) event).setParent(eventParent);
-
-    assertThat(event.getCorrelationId(), is(CORRELATION_ID));
-  }
-
-  @Test
-  public void noCorrelationIdInContextSequence() {
-    final MuleMessage message = MuleMessage.builder().payload(TEST_PAYLOAD).build();
-    final MuleEvent event =
-        MuleEvent.builder(executionContext).message(message).flow(flow).correlation(new Correlation(null, 6)).build();
-
-    assertThat(event.getCorrelationId(), is(MSG_EXEC_CTX_ID + ":" + "6"));
-  }
-
-  @Test
-  public void parentNoCorrelationIdInContextSequence() {
-    final MuleMessage messageParent = MuleMessage.builder().payload(TEST_PAYLOAD).build();
-    final MuleMessage message = MuleMessage.builder().payload(TEST_PAYLOAD).build();
-    final MuleEvent eventParent = MuleEvent.builder(executionContext).message(messageParent).flow(flow).build();
-    ((DefaultFlowCallStack) eventParent.getFlowCallStack()).push(new FlowStackElement(flow.getName(), "/0"));
-    final MuleEvent event =
-        MuleEvent.builder(executionContext).message(message).flow(flow).correlation(new Correlation(null, 6)).build();
-    ((DefaultMuleEvent) event).setParent(eventParent);
-
-    assertThat(event.getCorrelationId(), is(MSG_EXEC_CTX_ID + ":" + "/0" + ":" + "6"));
-  }
-
-  @Test
-  public void correlationIdInContextSequence() {
```

Comment:
It was testing that correlationId included sequence id composed in it though.. which we don't want to do and is no longer valid.